### PR TITLE
M3 final touches

### PR DESCRIPTION
**Pull request by ahzia**
In this milestone:

- added basic styling to my leaderboard Project.
- added a message to show the loading when a request has been made to API.
- added the local Storage Support to store the game ID after the first time load(after the game has been created).

Please review it and provide your feedback.

### CODE REVIEW

File: src/index.html [hunk 1]
```diff
@@ -4,23 +4,25 @@
...
+                <p>Add Your score</p>
+                <input type='text' id='name' placeholder='Your name' required>
+                <input type='text' id='score' placeholder='Your Score' required>
+                <input type='submit' id='submit' value='Submit'>
+                <p id='error' class='error'></p>
+            </form>
             <p>Recent Scores</p>
             <button id='refresh'>Refresh</button>
-            <div id='leaderboard'></div>
+            <div id='leaderboard' class='leaderboard'></div>
```
Comment (ZawadiSumba66) [inline src/index.html]: - [x] Kindly lets follow the layout of the wireframe given 🙏 . You can change the colors, background, fonts etc. But the positioning of the elements should be the same as from the requirements: 👇 
image

image

**Review by ZawadiSumba66 — CHANGES_REQUESTED** [review]
Hi  @ahzia 👋 , 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._
N/A

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by Georjane — APPROVED** [review]
## Status: PR Approved 💯 🥇 
#### Hello @ahzia , 

Your project is complete!  You have met all requirements for this project. It is a thumbs up for me. 👍🏽 You can go ahead and merge this PR. 
Congratulations! 🎉 👏👏

_Good luck with your next project and Happy coding!_ 💻

File: src/index.html (hunk 1 above)
Comment (ahzia) [reply src/index.html]: Thanks for the Review, I implemented your feedback.

**Comment by ahzia** [comment]
@Georjane Thanks for the Review

**Pull request merged**